packages/jest-reporters/src/__tests__/GithubActionsLogsReporter.test.js

```diff
@@ -0,0 +1,414 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
```

Comment:
Perhaps tests from this file could live inside already existing `GitHubActionsReporter.test.ts`?